src/cls/queue/cls_queue_src.cc

```diff
@@ -67,12 +67,12 @@ int queue_read_head(cls_method_context_t hctx, cls_queue_head& head)
     return -EINVAL;
   }
 
-  uint8_t decoded_head_size = sizeof(uint64_t) + sizeof(uint16_t);
+  constexpr auto decoded_head_size = sizeof(queue_head_start) + sizeof(encoded_len);
```

Comment:
is this the operative part of the bugfix?  might be useful to add a comment here

Reply:
no, just style issue. the bug fix is in line 72
if head is more than 1K, we should start from the 1K offset
before the fix, we were starting from the 10 byte offset (=decoded_head_size)